gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -624,20 +629,23 @@ def __init__(self,
                file_pattern,
                compression_type=CompressionTypes.AUTO,
                buffer_size=DEFAULT_VCF_READ_BUFFER_SIZE,
-               validate=True):
+               validate=True,
+               allow_failures=False):
```

Comment:
s/allow_failures/allow_malformed_records ("failure" is too broad).

Reply:
Done.